recipe/meta.yaml

```diff
@@ -36,23 +39,29 @@ requirements:
     - pillow >=3.1.1
     - pyct >=0.4.5
     - scipy
-    - toolz >=0.7.4
     - xarray >=0.9.6
+  run_constrained:
```

Comment:
I'm a bit confused by the use of `run_constrained` here for `numpy`. My understanding of `run_constrained` is that it is [used to specify an optional run dependency (and apply additional constraints to them)](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained). But in this case `numpy` is a run requirement, **_and_** it's being specified again under `run_constrained`. Could you please explain why?

Reply:
@pseudoyim Datashader's `test/imports` fail with `ImportError: Numba needs NumPy 1.21 or less`. I guessed that I can test it with `numpy <1.21` in the test environment but then in `run` always will be the same error for other users.
I decided to check on how `numba` is related to `numpy` and found this line in `run_constrained` in the `numba` recipe https://github.com/AnacondaRecipes/numba-feedstock/blob/7e739154a2a49e584df2cbc5b78b77bf61aa5cc4/recipe/meta.yaml#L65 
That's why I also use `run_constrained` for `numpy`. `Numba` uses `numpy` in `host`, `{{ pin_compatible('numpy') }}` in `run`, and n`umpy >=1.18,<1.22` in `run_constrained`. Without the last one in the `datashader` recipe the above-mentioned error will happen.

Reply:
@katietz Could you please verify if I use `run_constrained` correctly?